### ISSUE TITLE
Possible corpus with time information

### ISSUE THREAD

**Issue by piskvorky** [ashishbaghudana/dtm#1]
PubMed contains scholarly articles, including their publication date. A part of PubMed is public:
http://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/

It contains about a million documents, all from the medical domain. CC @tmylk.

**Comment by ashishbaghudana** [comment]
I did consider using this briefly. But the size of the corpus was rather inhibiting - I wasn't quite sure how I would preprocess and (for now) load into memory over 9GB of data.

**Comment by ashishbaghudana** [comment]
Other possible corpora I can find that might be useful:
- [BMC Corpora](http://www.biomedcentral.com/about/datamining) - A list of all OpenAccess Biomed Central journals: ~1.8 GB compressed, 11.6 GB uncompressed.
- [AG's corpus of news articles](http://www.di.unipi.it/~gulli/AG_corpus_of_news_articles.html) - A corpus of a million news articles.
- [British Parliament Debates from 1803](http://www.hansard-archive.parliament.uk/)
- [Leipzig Corpora Collection](http://corpora2.informatik.uni-leipzig.de/download.html) - 1 million sentences of text from newspapers for the each of the years 2005, 2006, 2007, 2008, 2009, 2010 and 2013 - however not separated by category or month.

**Comment by tmylk** [comment]
There is a paper on DTM applied to British Parliament Debates. We could
contact the authors for advice on pre-processing that they did in Python.

http://arxiv.org/pdf/1510.03797v1.pdf

On Sat, Nov 14, 2015 at 10:26 AM, Ashish Baghudana <notifications@github.com

**Comment by ashishbaghudana** [comment]
Yes, I'm just obtaining the entire dataset right now, I'll preprocess it at least according to year and place them in separate folders and push it here.
